scss/_mixins.scss

```diff
@@ -146,17 +91,4 @@
 */
 @mixin callout-variant($color) {
   border-left-color: $color;
-
```

Comment:
Not sure about this part, but took it out anyway.

Reply:
This part makes sure that the Callouts have the matching text color as the border. And anything using an inaccessible/light color (like Sky) uses the Dark Silver heading color. So with this change, all the headings are Dark Silver instead of the correct color.

Reply:
Thanks I've added back